course/models/models.py

```diff
@@ -162,6 +162,11 @@ def has_view_permission(self, user):
     def has_edit_permission(self, user):
         return user.is_teacher
 
+    def get_duplicate_question(self):
+        dupe_q = self
+        dupe_q.pk = None
+        return dupe_q
+
 
 class VariableQuestion(Question):
```

Comment:
here you should override the clone method to use the inherited clone method and additionally set the question_ptr_id to None. In this way, each question can have its own clone method that works with its inheritance structure even if it changes in the future.